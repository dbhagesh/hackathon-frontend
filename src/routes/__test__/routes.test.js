import { render } from "@testing-library/react";
import Routes from "../index";

// Test suite fot Phone Number Component
describe("<Layout /> ", () => {
  let wrapper;
  let props = {
    children: <></>,
  };

  /**
   *  Runs before each test case execution usally done init code for
   *  test cases refer this https://jestjs.io/docs/en/setup-teardown
   * */
  beforeEach(() => {
    wrapper = render(<Routes {...props} />);
  });

  /**
   *  Test cases start here
   */
  it("should renders", () => {
    expect(wrapper).not.toBeNull();
  });

  it("should match Main snapshot", () => {
    wrapper = render(<Routes {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
