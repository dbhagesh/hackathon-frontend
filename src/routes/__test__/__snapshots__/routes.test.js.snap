// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main />  should match Main snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      id="modalRoot"
    />
    <div>
      <div
        class=" main"
      >
        <div
          class="innerContainer whiteBg flex-row"
        >
          <div
            class="messageContainer flex-col"
          >
            <div
              class="heading"
            >
              Sorry, this page is not available
            </div>
            <div
              class="subText"
            >
              <div>
                We are sorry, the page you requested for has not been found.
              </div>
              <div>
                Please go back to the Home page
              </div>
            </div>
            <div
              class="btn"
            >
              <a
                class="link int "
                href="/v3"
              >
                Next
                 
                <i
                  class=" undefined-icon undefined-icon-arrow_forward icon"
                />
              </a>
            </div>
          </div>
          <div
            class="imageContainer"
          >
            <img
              alt="not found"
              src="error.png"
            />
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        class=" main"
      >
        <div
          class="innerContainer whiteBg flex-row"
        >
          <div
            class="messageContainer flex-col"
          >
            <div
              class="heading"
            >
              Sorry, this page is not available
            </div>
            <div
              class="subText"
            >
              <div>
                We are sorry, the page you requested for has not been found.
              </div>
              <div>
                Please go back to the Home page
              </div>
            </div>
            <div
              class="btn"
            >
              <a
                class="link int "
                href="/v3"
              >
                Next
                 
                <i
                  class=" undefined-icon undefined-icon-arrow_forward icon"
                />
              </a>
            </div>
          </div>
          <div
            class="imageContainer"
          >
            <img
              alt="not found"
              src="error.png"
            />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class=" main"
    >
      <div
        class="innerContainer whiteBg flex-row"
      >
        <div
          class="messageContainer flex-col"
        >
          <div
            class="heading"
          >
            Sorry, this page is not available
          </div>
          <div
            class="subText"
          >
            <div>
              We are sorry, the page you requested for has not been found.
            </div>
            <div>
              Please go back to the Home page
            </div>
          </div>
          <div
            class="btn"
          >
            <a
              class="link int "
              href="/v3"
            >
              Next
               
              <i
                class=" undefined-icon undefined-icon-arrow_forward icon"
              />
            </a>
          </div>
        </div>
        <div
          class="imageContainer"
        >
          <img
            alt="not found"
            src="error.png"
          />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
